ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouterMetrics.java

```diff
@@ -88,7 +191,7 @@ public Long getValue() {
         return chunkFillerThread.isAlive() ? 1L : 0L;
       }
     };
-    metricRegistry.register(MetricRegistry.name(NonBlockingRouter.class, chunkFillerThread.getName() + "-Running"),
+    metricRegistry.register(MetricRegistry.name(NonBlockingRouter.class, chunkFillerThread.getName() + "Running"),
```

Comment:
How will you differentiate the regex in sensors for gauges for tracking live request response handler threads and chunk filler threads? Looks like both of them have ".*Running".